src/components/Button/Button.props.ts

```diff
@@ -22,6 +22,11 @@ import { HTMLType, Hierarchy, IconPosition, Shape, Size, Type } from './Button.t
 
 export type ButtonProps = {
 
+  /**
+   * Fits button width to its parent width
+   */
+  block?: boolean,
```

Comment:
boolean prop should use the format "isBlock", "canBlock" etc (e.g. isVisible instead of visible)

This rule has never been written yet, we should add it to the contributing guidelines

Reply:
could it be "mustFitParent" ?

Reply:
I think "isBlock" is fine